packages/radix-ui-themes/package.json

```diff
@@ -137,7 +137,7 @@
   "devDependencies": {
     "@types/react": "^18.2.53",
     "@types/react-dom": "^18.2.18",
-    "autoprefixer": "^10.4.16",
+    "autoprefixer": "^10.4.19",
```

Comment:
Changes since 10.4.16:

<img width="1170" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/9430b8ea-4236-41d5-b0ee-a526e838c4e3">